css-frontend/src/main/java/org/sonar/css/model/property/standard/ColumnCount.java

```diff
@@ -22,10 +22,13 @@
 import org.sonar.css.model.property.StandardProperty;
 import org.sonar.css.model.property.validator.ValidatorFactory;
 
+import java.lang.System;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary import: java.lang classes are always implicitly imported. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)